### DISCUSSION TITLE
No more s_v_web_id cookie in Chrome ?

### DISCUSSION THREAD

**Discussion by syrom** [davidteather/TikTok-Api#810] [Q&A]
Hi David,

I only use your API occasisionally - but if I do, it comes in really handy. So HUGE thanks for that.

However, today I face the issue for the first time that, after logging into the web version of TikTok, Chrome won't show me any s_v_web_id cookie - and hence I can't run the code with a valid custom_verifyFp parameter: 
![TikTok_cookies](https://user-images.githubusercontent.com/8476842/151558990-8b08609b-a466-468c-ad42-43df2a62678e.png)

Are these structural changes they implemented, impacting your API - or just bad luck ?

I checked the values of the different cookies - but none looked similar to what the old content of the s_v_web_id cookie would look like.

Thanks in advance for your return,

Ulf

**Comment by juppcamper** [comment]
I face the exact same problem!

**Comment by soujh** [comment]
Same ! Is there an other way ?

**Reply by juppcamper** [reply]
The strange thing: I have the problem only under macOS, under Windows it is still displayed to me. Which OS do you use?

**Reply by soujh** [reply]
I tried to display  it on Windows and MacOs using Google Chrome. This does not appear on either.

**Comment by davidteather** [comment]
Using msToken in V6
